.gitlab-ci.yml

```diff
@@ -217,10 +215,17 @@ build:edge+flambda:
     COQ_EXTRA_CONF: "-native-compiler yes -coqide opt -flambda-opts "
     COQ_EXTRA_CONF_QUOTE: "-O3 -unbox-closures"
 
-build:egde:dune:
+build:egde:dune:dev:
   <<: *dune-template
   variables:
     OPAM_SWITCH: edge
+    DUNE_TARGET: world
+
+build:egde:dune:release:
+  <<: *dune-template
+  variables:
+    OPAM_SWITCH: edge
+    DUNE_TARGET: release
```

Comment:
The number of different build jobs is becoming a bit too much :(

Reply:
This would be the seed for the OPAM packaging job, it could be moved along the `pkg:nix` job tho.

Reply:
In fact the system is ready to generate the OPAM packages now, but I was planning to wait to:
- add documentation build target
- add test-suite build target
- fix the couple of bugs around [such as coqide path detection]
- wait for the better version number system to land in Dune.

Reply:
Fine by me to do this in several steps. But indeed, do move this next to the pkg:nix job. If fact, once the documentation / test-suite targets are ready, we could even use this to build the Nix package.

Reply:
Ok, moving to `pkg:dune-release`